python/setup.py.in

```diff
@@ -10,15 +10,15 @@ netconf2Module = Extension("netconf2",
                                    ],
                            depends=["${CMAKE_CURRENT_SOURCE_DIR}/netconf.h",
                                     "${CMAKE_CURRENT_SOURCE_DIR}/session.h",
-                                    "${CMAKE_CURRENT_COURCE_DIR}/rpc.h"
+                                    "${CMAKE_CURRENT_SOURCE_DIR}/rpc.h"
```

Comment:
This is a general Error and should be fixed anyway!

Reply:
Yes, seems that way.